### DISCUSSION TITLE
Attribute in the note

### DISCUSSION THREAD

**Discussion by marco10x15** [TriliumNext/Trilium#3656] [Q&A]
Hello, I've discovered this beautiful software and I'm in the process of learning to use it.
I have some question, that i put in different post in order to not messy the answers.

I've seen the “Word count widget” example that is possible using Javascript view the word count and I'm thinking that can be possible to have a similar widget to display the “owned attribute” of my interest (dropping) just under the “note menu”?

And second step to print  this and not the raw attribute?

I've zero knowledge of JavaScript (begin to read something but I don't think to be able to do that by my own).

Thanks!

**Comment by zadam** [comment]
Yes, custom widgets can be used for that, but you need JavaScript/HTML/CSS coding skills for that.

**Reply by marco10x15** [reply]
Thanks @zadam.
Now, there is someone in the forum with skill and patience to help me with the task of creating this feature?
